src/quo2/components/avatars/group_avatar/view.cljs

```diff
@@ -7,24 +7,27 @@
 
 (def sizes
   {:icon      {:small  12
+               28      16
                :medium 16
                :large  20}
    :container {:small  20
+               28      28
                :medium 32
                :large  48}})
 
 ;; TODO: this implementation does not support group display picture (can only display default group
 ;; icon).
 (defn- view-internal
   [_]
```

Comment:
@vkjr I agree, but I'm not touching this component more than required, if I'd refactor this component it would imply even more changes and this PR is long enough.

as I said in this PR description:
> So I implemented them in order to have this PR working and pass design review, but these components should be checked in the same way I did it with this one.

So I think a specific issue should be created to fix all these problems.

Reply:
@ulisesmac, I see, no worries) I just comment everything that is close enough to the changes you've made :))